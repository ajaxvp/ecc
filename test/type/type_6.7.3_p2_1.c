/* ISO: 6.7.3 (2); restrict-qualification constraint */

char *restrict a;
restrict char b;
