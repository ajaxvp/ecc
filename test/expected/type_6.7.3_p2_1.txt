ecc: error: [4:1] type must be a pointer to an object or incomplete type in order to be restrict-qualified
